Base/QTCore/qSlicerCorePythonManager.h

```diff
@@ -50,6 +50,9 @@ class Q_SLICER_BASE_QTCORE_EXPORT qSlicerCorePythonManager : public ctkAbstractP
   /// Append \a paths to \a sys.path
   void appendPythonPaths(const QStringList& paths);
 
+  // /// Convert a string to a safe python string literal
```

Comment:
Keep only 3 slashes instead of 5.
Describe which quote is used for the literal (`"` or `'`). Probably `'` would be better (later quote character can be made configurable).